android/src/com/frostwire/android/gui/activities/MainActivity.java

```diff
@@ -676,7 +577,7 @@ private void showLastBackDialog() {
         dlg.show(getFragmentManager()); //see onDialogClick
     }
 
-    private void showShutdownDialog() {
+    public void showShutdownDialog() {
```

Comment:
which I had `friend` classes in Java.
If I understand correctly, I would still leave this private, but the friend, in this case `MainController` could have access to this method.